src/ask-password/ask-password.c

```diff
@@ -155,6 +165,18 @@ static int parse_argv(int argc, char *argv[]) {
                         assert_not_reached("Unhandled option");
                 }
 
+        if (!emoji || streq(emoji, "auto"))
+                SET_FLAG(arg_flags, ASK_PASSWORD_NO_EMOJI, arg_flags & ASK_PASSWORD_ECHO);
```

Comment:
Uh, again the condition seems inverted... Maybe,
```
SET_FLAG(arg_flags, ASK_PASSWORD_NO_EMOJI, !FLAGS_SET(arg_flags, ASK_PASSWORD_ECHO));
```
So, this is why I prefer to invert the meaning of the flag...

Reply:
No, it is not inverted...

But I will update to change the meaning. 😝

Reply:
Ah, yeah, sorry. `which defaults to yes _unless_ --echo is given`. I forgot `_unless_`.